postcss.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  plugins: [
+    require('autoprefixer')
+  ]
+}
```

Comment:
I don't think that this is being used via the `postcss-loader`. I think instead we need to config the loader in `webpack.config.js` via something like:

```js
{
  loader: 'postcss-loader',
  options: {
    autoprefixer: {
      browsers: ['last 2 versions'],
    },
    plugins: () => [autoprefixer],
  },
}
```